vividus-plugin-kafka/src/main/java/org/vividus/steps/kafka/KafkaSteps.java

```diff
@@ -113,19 +115,42 @@ private <T> Map<String, T> convert(String propertiesPrefix, IPropertyParser prop
     }
 
     /**
-     * Send the event with the specified value to the provided topic with no key or partition.
+     * Sends the event with the key and the value to the provided topic. Events with the same event key are written to
```

Comment:
it doesn't make sense to put info about event keys to step which doesn't accept key

Reply:
ops, it had to be the step that accepts key :)